doc/cygnus-common/installation_and_administration_guide/management_interface_v1.md

```diff
@@ -1011,6 +1015,8 @@ Additionally, because Cygnus distributes event processing among sources (respons
 * The service time is computed as the total sum of processing times (understanding processing time in this context as the time between even reception in the source and processing in one of the invidual sinks) divided by the number of times the same incoming event is persisted (in other words, it is not divided by the number of incoming transactions). Thus, it could occur N events are persisted M times (M > N), and that impacts in the average service time.
 * The same occurs with the number of transaction errors. Thus, it could be this number is greater than the number of incoming transactions because the events are being processed by 2 or more sinks, and all of them are failing.
 
+Finally, because Cygnus implements a retry mechanism for those persistence operations that failed, it must be noticed the number of outgoing transactions (and its related metrics) may be much higher than the number of icoming transactions (even when a single a sink is used).
```

Comment:
"icoming" issue fixed in https://github.com/telefonicaid/fiware-cygnus/pull/1385/commits/07a7b2349d27c6801078b2717cdd83c0a9bac56e